plugin/documents.py

```diff
@@ -228,15 +228,25 @@ def _on_code_actions(self, responses: CodeActionsByConfigName) -> None:
         if action_count == 0:
             return
         suffix = 's' if action_count > 1 else ''
-        code_actions_link = make_link('subl:lsp_code_actions', '{} code action{}'.format(action_count, suffix))
-        self.view.add_regions(self.ACTIONS_ANNOTATION_KEY,
-                              [sublime.Region(self._stored_region.b, self._stored_region.a)],
-                              flags=sublime.DRAW_NO_FILL | sublime.DRAW_NO_OUTLINE,
-                              annotations=["<div class=\"actions\">{}</div>".format(code_actions_link)],
-                              annotation_color='#2196F3')
+        flags = sublime.DRAW_NO_FILL | sublime.DRAW_NO_OUTLINE
+        region = sublime.Region(self._stored_region.b, self._stored_region.a)
+
+        if global_settings.show_code_actions == 'annotation':
+            code_actions_link = make_link('subl:lsp_code_actions', '{} code action{}'.format(action_count, suffix))
+            self.view.add_regions(self.CODE_ACTIONS_KEY,
+                                  [region],
+                                  flags=flags,
+                                  annotations=["<div class=\"actions\">{}</div>".format(code_actions_link)],
+                                  annotation_color='#2196F3')
+        elif global_settings.show_code_actions == 'bulb':
+            self.view.add_regions(self.CODE_ACTIONS_KEY,
+                                  [region],
+                                  'markup.changed',
+                                  'Packages/LSP/icons/lightbulb.png',
+                                  flags)
```

Comment:
There are a few duplicated arguments for the `add_regions` function here. I would prefer if this was a single call to `add_regions`, like so:

```python
if global_settings.show_code_actions == 'annotation':
    ...
elif global_settings.show_code_actions == 'bulb':
    ...
else:
    # handle this case!
    pass
self.view.add_regions(self.CODE_ACTIONS_KEY, [region], ...)
```

Reply:
Done with cb2a51e